### PR TITLE
Create ui-c8y-1020-0-0-add-info-on-how-to-style-shell-application-via-plugin-6177.md

### PR DESCRIPTION
**Pull request by gen-App-Enablement**
Release note created by daj

Ticket: [MTM-58246](https://cumulocity.atlassian.net/browse/MTM-58246)
Original PR: [6177](https://github.softwareag.com/IOTA/cumulocity-ui/pull/6177)

### CODE REVIEW

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-add-info-on-how-to-style-shell-application-via-plugin-6177.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
+---
+date: ""
+title: add info on how to style shell application via plugin (#6177)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-add-info-on-how-to-style-shell-application-via-plugin-6177.md:3]: The Claude AI suggested to use the following title:
```suggestion
title: Add information on how to style the shell application using a plugin
```

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-add-info-on-how-to-style-shell-application-via-plugin-6177.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58246
+version: 1020.0.0
+---
+add info on how to style shell application via plugin (#6177)
```
Comment (gen-App-Enablement) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-add-info-on-how-to-style-shell-application-via-plugin-6177.md:17]: The Claude AI suggested to use the following body:
```suggestion
To provide more flexibility in customizing the look and feel of the shell application, a new plugin mechanism has been introduced. This change allows developers to create plugins that can modify the styling of various elements in the shell application. The impact of this change is that developers now have more control over the visual appearance of the shell application and can tailor it to their specific needs or branding requirements without having to modify the core application code.
```

**Review by gen-App-Enablement — COMMENTED** [review]
AI improvements

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-add-info-on-how-to-style-shell-application-via-plugin-6177.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+component:
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58246
+version: 1020.0.0
+---
+Documentation has been added on how to style the shell application via plugins.
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-add-info-on-how-to-style-shell-application-via-plugin-6177.md:17]: @DawidSAG Can you please providedetails where it has been added?

Comment (DawidSAG) [reply content/change-logs/application-enablement/ui-c8y-1020-0-0-add-info-on-how-to-style-shell-application-via-plugin-6177.md:17]: @BeateRixen can I just add link to the codex? https://styleguide.cumulocity.com/apps/codex/#/getting-started/guides/theming/overview#styling-the-shell-application-with-a-plugin
```
Documentation has been added on how to style the shell application via plugins. 
For details, refer to [{{< product-c8y-iot >}} Codex](https://styleguide.cumulocity.com/apps/codex/#/getting-started/guides/theming/overview#styling-the-shell-application-with-a-plugin).
```

Comment (BeateRixen) [reply content/change-logs/application-enablement/ui-c8y-1020-0-0-add-info-on-how-to-style-shell-application-via-plugin-6177.md:17]: Yes, exactly :-)

**Review by DawidSAG — APPROVED** [review]

File: content/change-logs/application-enablement/ui-c8y-1020-0-0-add-info-on-how-to-style-shell-application-via-plugin-6177.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+  - value: component-YbYJ3gLU_
+    label: Web SDK
+build_artifact:
+  - value: tc-pjJiURv9Y
+    label: ui-c8y
+ticket: MTM-58246
+version: 1020.0.0
+---
+Documentation has been added on how to style the shell application via plugins. 
+For details, refer to [{{< product-c8y-iot >}} Codex](https://styleguide.cumulocity.com/apps/codex/#/getting-started/guides/theming/overview#styling-the-shell-application-with-a-plugin).
```
Comment (BeateRixen) [inline content/change-logs/application-enablement/ui-c8y-1020-0-0-add-info-on-how-to-style-shell-application-via-plugin-6177.md:18]:
```suggestion
For details, refer to the [{{< product-c8y-iot >}} Codex](https://styleguide.cumulocity.com/apps/codex/#/getting-started/guides/theming/overview#styling-the-shell-application-with-a-plugin).
```

**Pull request merged**